packages/@docs/demos/src/demos/core/Menu/Menu.demo.disabledTooltip.tsx

```diff
@@ -0,0 +1,79 @@
+import {
```

Comment:
Please remove this file, it is not needed.

Reply:
@rtivital 
Thank you for your review.
We have addressed your advice.